site/docs/styles.css

```diff
@@ -66,3 +66,78 @@
     fill: blue;
   }
 }
+
+.vocs_Content:has(#blobs) {
+  padding: 60px 0px 0px 0px;
+}
+
+.dark .vocs_Content:has(#blobs) {
+  padding: 60px 0px 0px 0px;
+  background-color: #030713;
+}
+
+.vocs_Content:has(#blobs) + .vocs_Footer {
+  display: none;
+}
+
+.dark .css-main-container {
+  background-color: #111827;
+}
+
+.css-main-container {
+  background-color: #f3f4f6;
+}
+
+.css-alternate-container {
+  background-color: white;
+}
+.dark .css-alternate-container {
+  background-color: #030713;
+}
+
+#landing-page .vocs_Header,
```

Comment:
@abcrane123 oh! very ancient rule about CSS. never, and I say never use `#` as selector. It has some weird behaviors of oveeriding which is not convinient to use.

Instead add an extra CSS class.